crates/utils/src/tracing/grpc.rs

```diff
@@ -1,16 +1,25 @@
-use tracing_opentelemetry::OpenTelemetrySpanExt;
+/// Creates a [`tracing::Span`] based on RPC service and method name.
+macro_rules! rpc_span {
```

Comment:
This seemed like the best way of feeding in the rpc.* attributes into the span. Please double check my usage of `info_span!` w.r.t the attributes.

Reply:
I'm actually wondering - what does the full rpc request path look like? 

Maybe we can avoid the path splitting and macro entirely if the path is good enough?

As in, the path might be `xxxx/store.api/{method}`

Reply:
Ah wait. The issue is that `tracing::span_info!` requires static strings right?

Hmm. Okay so there isn't really a way to do this properly for the span name itself since you cannot do it dynamically.